rust/ddlog_benches/Makefile.toml

```diff
@@ -1,6 +1,6 @@
 # Runs all ddlog benchmarks
 [tasks.benchmarks]
-dependencies = ["download-data", "bench-twitter", "bench-livejournal"]
+dependencies = ["bench-twitter", "bench-livejournal"]
```

Comment:
What's the reason for this change?

Reply:
without this change 'cargo bench' fails.

Reply:
What's the error message?